cmd/osv-scanner/__snapshots__/main_test.snap

```diff
@@ -364,13 +365,16 @@ overriding license for package Packagist/league/flysystem/1.0.8 with 0BSD
 +-------------------------------------+------+-----------+------------------+---------+---------------------------------------+
 | https://osv.dev/GHSA-9f46-5r25-5wfm | 9.8  | Packagist | league/flysystem | 1.0.8   | fixtures/locks-insecure/composer.lock |
 +-------------------------------------+------+-----------+------------------+---------+---------------------------------------+
-+-------------------+-----------+------------------+---------+---------------------------------------+
-| LICENSE VIOLATION | ECOSYSTEM | PACKAGE          | VERSION | SOURCE                                |
-+-------------------+-----------+------------------+---------+---------------------------------------+
-| 0BSD              | Packagist | league/flysystem | 1.0.8   | fixtures/locks-insecure/composer.lock |
-| UNKNOWN           | RubyGems  | ast              | 2.4.2   | fixtures/locks-many/Gemfile.lock      |
-| 0BSD              | Packagist | sentry/sdk       | 2.0.4   | fixtures/locks-many/composer.lock     |
-+-------------------+-----------+------------------+---------+---------------------------------------+
++-------------------+-----------+------------------------------------------------+---------+-------------------------------------------------------+
+| LICENSE VIOLATION | ECOSYSTEM | PACKAGE                                        | VERSION | SOURCE                                                |
++-------------------+-----------+------------------------------------------------+---------+-------------------------------------------------------+
+| 0BSD              | Packagist | league/flysystem                               | 1.0.8   | fixtures/locks-insecure/composer.lock                 |
+| UNKNOWN           |           | https://github.com/flutter/buildroot.git       |         | fixtures/locks-insecure/osv-scanner-flutter-deps.json |
+| UNKNOWN           |           | https://github.com/brendan-duncan/archive.git  |         | fixtures/locks-insecure/osv-scanner-flutter-deps.json |
+| UNKNOWN           |           | https://chromium.googlesource.com/chromium/src |         | fixtures/locks-insecure/osv-scanner-flutter-deps.json |
```

Comment:
note: I realised as part of this that git-based dependencies are a bit of an edge case for the more composition-based config override logic

internally they don't have an ecosystem which is reflected in the license violation table but the vulnerability table uses `GIT` instead, but then `ecosystem = "GIT"` does not match these while `ecosystem = ""` just ignores _everything_.

I don't know if its worth fixing but we could account for this by mapping a value of `GIT` as a special case when checking for overrides; alternatively (and maybe the better short-term change) we could change the vulnerabilities table output to not have this logic so it at least matches the licenses table...